EIPS/eip-7742.md

```diff
@@ -44,37 +44,68 @@ and deployment of changes to the blob count easier.
+
+#### Normalized excess blob gas
+
+At any block we don't want the gas update factor (whether pushing gas up or down) to exceed ~1.125. This is essentially achieved in [EIP-4844](./eip-4844.md) by having an appropriate `BLOB_BASE_FEE_UPDATE_FRACTION` to divide `excess_blob_gas` accumulator while exponetiating it
+
+Since the target from CL could theoretically change block to block, we need to normalize excess gas accumulation because the update fraction is actually directly based on both the target and max (max value `excess_blob_gas` can jump by in a block).
+
+We propose to "normalize" the gas diffs every block against its target/max and then accumulate in `excess_blob_gas` so that we accumulate normalized `excess_blob_gas` every block and can use a constant normalized update fraction in our fee calculations.
 
 ## Specification
 
-### Block structure and validity
+### Execution Block structure and validity
 
 Upon activation of this EIP, execution clients **MUST** extend the header schema with an
-additional 64-bit field: the `target_blob_count`. This value is set to the current target blob count. The Engine API
-is modified along with this EIP to provide the `target_blob_count` with each payload and implementations can use this
-value to correctly set the block header field.
+additional 64-bit field: the `target_blob_count`. This values is set to the one provided by the CL with the modified engine api fcU's payload attributes.
+
+Validity of these values is guaranteed from the consensus layer, much like how withdrawals are handled. Hence when verifying a block, execution clients **MUST** ensure the target blob count in the block header matches the one provided by the consensus client.
+
+For a genesis block with no existing parent, the value should be set according to the agreed specification for the target blob count given by that genesis block's protocol rule set.
+
+`excess_blob_gas` in the block now holds _normalized_ excess blob gas accumulating `((blob_gas_used-target_blob_gas)/target_blob_gas) * EXCESS_BLOB_GAS_NORMALIZATION_FACTOR`. We choose `EXCESS_BLOB_GAS_NORMALIZATION_FACTOR` to `100E6` and this makes our choice of `NORMALIZED_BLOB_BASE_FEE_UPDATE_FRACTION` as `3338477 * 100E6 // 393216 = 849018605`.
+
+| Constant | Value |
+| - | - |
+| `EXCESS_BLOB_GAS_NORMALIZATION_FACTOR` | `100E6` |
+| `NORMALIZED_BLOB_BASE_FEE_UPDATE_FRACTION` | `849018605` |
+| `OLD_TARGET_BLOB_GAS_PER_BLOCK` | `393216` |
+
+At the fork block (where parent.timestamp < FORK_TIMESTAMP), we convert parent's accumulated `excess_blob_gas` to the normalized `excess_blob_gas` to be used in this block's `excess_blob_gas` accumulator.
+
+```python
+def calc_excess_blob_gas(parent: Header) -> int:
+  if(parent.timestamp < FORK_TIMESTAMP)
+    normalizedParentExcessBlobGas = parent.excess_blob_gas * EXCESS_BLOB_GAS_NORMALIZATION_FACTOR // OLD_TARGET_BLOB_GAS_PER_BLOCK
```

Comment:
would be nice if all of these `camelCase` symbols were `snake_case`, just to follow python conventions